app/assets/javascripts/common/reports.js

```diff
@@ -1693,15 +1695,25 @@ function baseBarChartConfig() {
 const dynamicChartSegementDashed = (
   ctx,
   numberOfXAxisTicks,
```

Comment:
`numberOfXAxisTicks` does this return the number of months on the x-axis?

Reply:
they are the same figure, but I wanted to make it generic incase the x axis is not month strings.
I used the data point with the labels (for the x axis) but I could have also used the actual data array for the line. I just need to get the number of data points in the chart for the x-axis